mlir/lib/Dialect/SCF/IR/SCF.cpp

```diff
@@ -564,6 +566,8 @@ void ForOp::getSuccessorRegions(RegionBranchPoint point,
 
 SmallVector<Region *> ForallOp::getLoopRegions() { return {&getRegion()}; }
 
+OperandRange ForallOp::getInits() { return getOutputs(); }
```

Comment:
Actually, the "outputs" and bbarg of `scf.forall` is not an iter_arg like `scf.for`. Because there are no iterations, just parallelism. Maybe we want to treat them like iter_args at some point, but I'm removing it again for now.